Remove unused nodes cloumn

x11562 opened the pull request:
column "is_storage" and column "storage" are repeated, and column "is_storage" has no initial value, it is recommended to delete

MohamedBassem commented:
Hi @x11562 for sending this fix. Do you mind also removing `sequencer_enabled` as well? Doesn't seem to be used as well?

You'll probably need to run `logdevice/ops/ldquery/markdown-ldquery > docs/administration/ldquery.md` as well to update the documentation of this table.

x11562 commented:
> Hi @x11562 for sending this fix. Do you mind also removing `sequencer_enabled` as well? Doesn't seem to be used as well?
> 
> You'll probably need to run `logdevice/ops/ldquery/markdown-ldquery > docs/administration/ldquery.md` as well to update the documentation of this table.

Hi, @MohamedBassem  I have removed `sequencer_enabled`, and updated  ldquery.md
